client/src/test/scala/org/http4s/client/testroutes/GetRoutes.scala

```diff
@@ -4,16 +4,19 @@ import cats.effect._
 import fs2._
 import org.http4s.Response
 import org.http4s.Status._
+import cats.implicits._
 
 object GetRoutes {
   val SimplePath = "/simple"
   val ChunkedPath = "/chunked"
+  val DelayedPath = "/delayed"
 
   val getPaths: Map[String, Response[IO]] = {
     Map(
       SimplePath -> Response[IO](Ok).withBody("simple path"),
       ChunkedPath -> Response[IO](Ok).withBody(
-        Stream.emits("chunk".toSeq.map(_.toString)).covary[IO])
+        Stream.emits("chunk".toSeq.map(_.toString)).covary[IO]),
+      DelayedPath -> IO(Thread.sleep(1000L)) *> Response[IO](Ok).withBody("delayed path")
```

Comment:
We can sleep with an fs2 scheduler and not block a thread.  This should make tests more robust.